.travis.yml

```diff
@@ -20,6 +20,8 @@ install: travis_retry composer update --prefer-dist
 script:
   - vendor/bin/phpcs
   - vendor/bin/phpcs $(find tests/input/* | sort) --report=summary --report-file=phpcs.log; diff tests/expected_report.txt phpcs.log
+  - vendor/bin/phpstan analyse
```

Comment:
These should be put into separate stages like we do on other projects.

Reply:
Other Doctrine projects? I'll take a look into that.